woocommerce/woocommerce-ios
# [Woo POS] Update POS menu item based on the latest design

Part of #12779 


## Why

Now that we have the design for the POS menu item in p91TBi-beb-p2, we can update it in the app. Please note that more design updates might be coming based on the p2 comments, I added a subtask to #12779 to address them after the design is finalized.

## How

- Two new icons were added, `pointOfSaleImage` from the asset, and `playSquareImage` from SF symbols
- The title, description, icon updates of the POS menu item are in `HubMenuViewModel.PointOfSaleEntryPoint`
- For the `playSquareImage` in the accessory view of the POS row, a new case of `HubMenu.Row.Chevron` enum was added to show a play button in the accessory view instead of a chevron as the default. Originally I wanted to make it a configuration per row, but the enum is currently fileprivate in the view file and I decided to keep the changes minimum

## Testing instructions


Prerequisite: having a store eligible for POS

- Switch to a store in the prerequisite if needed
- Go to the Menu tab
- (If the POS row isn't shown yet) Turn on the feature switch in Menu > Settings > Experimental Features > POS, then revisit the Menu tab, if needed --> the POS row should look as in design

## Screenshots


light | dark
-- | --
<img src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/e49d6f13-c995-4d6e-b8db-4b17bf333ec9" width="300" /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/8f1bafc5-f6c0-4ff4-97d2-6347a5c403a7" width="300" />

---
